Cargo.toml

```diff
@@ -58,22 +58,10 @@ std = [
   "ring/std",
 ]
 
-tool = [
-  "inject-alloc",
```

Comment:
inject-alloc no longer appears in the tool's dependency list.  Is that intentional?

Reply:
It does, see
```
manticore = { path = "..", features = ["inject-alloc"] }
```

This is mostly an artifact of poor choices when originally structuring the tool binary. =)